docs/manual/resource-leak-checker.tex

```diff
@@ -0,0 +1,326 @@
+\htmlhr
+\chapterAndLabel{Resource Leak Checker for must-call obligations}{resource-leak-checker}
+
+The Resource Leak Checker guarantees that the program fulfills every object's
+must-call obligations before the object is de-allocated.
```

Comment:
This is intentional, to avoid overclaiming by saying something like "the Resource Leak Checker guarantees that there are no resource leaks in the program". That's not really true (because "resource leak" could refer to lots of kinds of resources, some of which might not have appropriate `@MustCall` annotations). I'm open to any suggestions you have that get across the idea of what the checker does without falling into that trap.